src/js/views/right_panel_view.js

```diff
@@ -1072,6 +794,8 @@
                     self.models.forEach(function(m){
                         m.save(to_save);
                     });
+                    // we don't listenTo zoom change...
```

Comment:
typo

Reply:
```listenTo()``` is a Backbone method, so it's referring to the fact that we don't call ```self.listenTo()```.

Reply:
that was not very clear so it will be better to be more explicit